docs/src/main/paradox/scala/http/06extensions.md

```diff
@@ -0,0 +1,3 @@
+# 6. Extensions ➕
```

Comment:
The extensions icon does not render here. Neither Chromium, nor IntelliJ. That is a rare plus you found. :)

Nevertheless, I like the icons. :)